src/views/community/modules/submit.php

```diff
@@ -0,0 +1,64 @@
+<?php
```

Comment:
Same question about only injecting the reCaptcha pieces rather than overriding the entire template.